be/src/olap/delta_writer.cpp

```diff
@@ -98,6 +100,8 @@ Status DeltaWriter::init() {
             MemTracker::create_tracker(-1, "DeltaWriter:" + std::to_string(_tablet->tablet_id()));
     // check tablet version number
     if (_tablet->version_count() > config::max_tablet_version_num) {
+        //trigger samll compaction
+        StorageEngine::instance()->submit_samll_compaction_task(_tablet);
```

Comment:
should we check whether to pass this check after small compaction?

Reply:
> should we check whether to pass this check after small compaction?

Small version merging is asynchronous, which is unlikely to be completed in a short time, so it is not necessary to check here